coc.html

```diff
@@ -43,40 +43,57 @@
+  <nav class="navbar navbar-custom navbar-fixed-top">
+    <div class="container">
+      <a class="navbar-brand index-logo" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
         <img src="./img/banner-logo.png" class="logo" alt="banner-logo" style="width: 150px; display: inline-block;">
       </a>
       <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      </button>
       </div>
 
-      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-        <ul class="nav nav-justified">
-          <li><a href="index.html#home">Home</a></li>
+     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav">
+          <li class="active"><a href="index.html#home">Home</a></li>
           <li><a href="index.html#journey">Journey</a></li>
-          <li><a href="index.html#speakers">Speakers</a></li>
-          <!-- <li><a href="#conference">Schedule</a></li> -->
+          <li><a href="index.html#">Schedule</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Speakers<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="index.html#speakers">Keynotes</a></li>
+              <li><a href="index.html#">All Speakers</a></li>
+            </ul>
+          </li>
           <li><a href="index.html#sponsors">Sponsors</a></li>
-          <li><a href="index.html#tickets">Tickets</a></li>
           <li><a href="https://in.pycon.org/blog/">Blog</a></li>
-          <li><a href="index.html#venue">Venue</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Attend<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="index.html#tickets">Tickets</a></li>
+              <li><a href="index.html#venue">Venue</a></li>
+              <li><a href="index.html#">Hotels</a></li>
+              <li><a href="index.html#">Traveling to Hyderabad</a></li>
+            </ul>
+          </li>
           <li><a href="faq.html">FAQ</a></li>
-          <li><a href="volunteer.html">Volunteer</a></li>
-          <li class="active"><a href="coc.html">COC</a></li>
+          <li class="dropdown">
```

Comment:
`<li class="active">` will be added here for the COC page contained in the more category.